### PR TITLE
T() macro 4

### PR DESCRIPTION
**Pull request by Dragorn421**
Sequel to https://github.com/zeldaret/oot/pull/2064

### CODE REVIEW

**Comment by fig02** [comment]
gc-eu-mq-dbg failed, for some reason

File: src/code/z_kankyo.c [hunk 1]
```diff
@@ -2103,9 +2103,9 @@ void Environment_PlaySceneSequence(PlayState* play) {
     }
 
     PRINTF("\n-----------------\n", ((void)0, gSaveContext.forcedSeqId));
-    PRINTF("\n 強制ＢＧＭ=[%d]", ((void)0, gSaveContext.forcedSeqId)); // "Forced BGM"
+    PRINTF(T("\n 強制ＢＧＭ=[%d]", "\n Forced BGM=[%d]"), ((void)0, gSaveContext.forcedSeqId));
     PRINTF("\n     ＢＧＭ=[%d]", play->sceneSequences.seqId);
```
Comment (krm01) [inline src/code/z_kankyo.c:2107]: missed this guy
```suggestion
    PRINTF(T("\n     ＢＧＭ=[%d]", "\n     BGM=[%d]"), play->sceneSequences.seqId);
```

File: src/code/z_sram.c [hunk 2]
```diff
@@ -856,7 +857,7 @@ void Sram_EraseSave(FileSelectState* fileSelect, SramContext* sramCtx) {
     MemCpy(sramCtx->readBuff + offset, &gSaveContext, sizeof(Save));
     SsSram_ReadWrite(OS_K1_TO_PHYSICAL(0xA8000000) + offset, &gSaveContext, SLOT_SIZE, OS_WRITE);
 
-    PRINTF("ＣＬＥＡＲ終了\n");
+    PRINTF(("ＣＬＥＡＲ終了\n", "CLEAR END\n"));
```
Comment (krm01) [inline src/code/z_sram.c:860]: forgor the T, this is why it failed
```suggestion
    PRINTF(T("ＣＬＥＡＲ終了\n", "CLEAR END\n"));
```

File: src/code/z_kankyo.c (hunk 1 above)
Comment (Dragorn421) [reply src/code/z_kankyo.c:2107]: I skip these fullwidth (https://en.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms) latin only strings but there's an argument to also "translate" them to ascii yeah
I'd rather do a separate PR for all of them for clarity and motivating it

File: src/code/z_demo.c [hunk 3]
```diff
@@ -202,7 +202,7 @@ void Cutscene_UpdateScripted(PlayState* play, CutsceneContext* csCtx) {
     }
 
     if ((gSaveContext.cutsceneTrigger != 0) && (csCtx->state == CS_STATE_IDLE)) {
-        PRINTF("\nデモ開始要求 発令！"); // "Cutscene start request announcement!"
+        PRINTF(T("\nデモ開始要求 発令！", "\nDemonstration start request issued!"));
         gSaveContext.save.cutsceneIndex = 0xFFFD;
```
Comment (fig02) [inline src/code/z_demo.c:206]: I think we should probably just translate it as "demo" instead of "demonstration". It is kind of its own word

Comment (Dragorn421) [reply src/code/z_demo.c:206]: Since we have (plenty) evidence they used "demo" for "cutscene" and the japanese is pronounced as "demo" I agree

Comment (fig02) [reply src/code/z_demo.c]: Yeah, the text is also just "demo" "デモ".
I assume google translate extrapolated on the abbreviation?

**Review by fig02 — APPROVED** [review]

**Review by krm01 — APPROVED** [review]

**Pull request merged**